pronotepy/dataClasses.py

```diff
@@ -952,48 +1012,145 @@ class Message(Object):
         the id of the message (used internally)
     author : str
         author of the message
-    recipients : list
-        Recipitents of the message. ! May be just ['# recipients'] !
     seen : bool
         if the message was seen
     date : datetime.datetime
         the date when the message was sent
+    content: str
+        content of the messages
     """
 
-    __slots__ = ["id", "author", "recipients", "seen", "date", "_client", "_listePM"]
+    __slots__ = ["id", "author", "seen", "date", "content", "_client", "_listePM"]
 
     def __init__(self, client: _ClientBase, json_dict: dict) -> None:
         super().__init__(json_dict)
         self._client = client
-        self._listePM = json_dict["listePossessionsMessages"]["V"]
 
         self.id: str = self._resolver(str, "N")
+        self.content: str = ""
         self.author: str = self._resolver(str, "public_gauche")
-        self.recipients: list = self._resolver(list, "listePublic")
-        self.seen: bool = self._resolver(bool, "lu")
-        self.date: datetime.datetime = self._resolver(
-            lambda d: Util.datetime_parse(" ".join(d.split()[1:])), "libelleDate"
+        self.seen: bool = self._resolver(bool, "lu", default=False)
+        self.date: datetime.datetime = self._resolver(Util.datetime_parse, "date", "V")
+
+        if self._resolver(bool, "estHTML", default=False):
+            self.content = self._resolver(Util.html_parse, "contenu", "V")
+        else:
+            self.content = self._resolver(str, "contenu")
+
+        del self._resolver
+
+    @classmethod
+    def get(cls, client: Client, id: str) -> Message:
+        """
+        Get the message of a specific id
+
+        id: str
+            id of the message
+
+        Returns
+        -------
+        Message
+        """
+        message = client.post(
+            "ListeMessages", 131, {"listePossessionsMessages": [{"N": id}]}
+        )
+
+        return Message(
+            client, message["donneesSec"]["donnees"]["listeMessages"]["V"][0]
+        )
+
+
+class Discussion(Object):
+    """
+    Represents a discussion.
+
+    Attributes
+    ----------
+    id : str
+        the id of the discussion (used internally)
+    subject: str
+        the subject of the discussion
+    creator: str
+        the person who open the discussion
+    messages: List[Message]
+        messages link to the discussion
+    unread: int
+        number of unread messages
+    close: bool
+        True if the discussion is close
+    date : datetime.datetime
+        the date when the discussion was open
+    """
+
+    __slots__ = ["id", "subject", "messages", "unread", "close", "date", "_client"]
+
+    def __init__(self, client: Client, json_dict: dict) -> None:
+        super().__init__(json_dict)
+        self._client = client
+        self._possessions: list = self._resolver(
```

Comment:
`self._possessions` is not in `__slots__` I'm not sure how it's letting you do this lmao

Reply:
I don't how work __slots__ it's very weird many times in my projects attributes are not in slots and it's work.